### PR TITLE
Add a spectrum scoring application egs_phd

### PR DESCRIPTION
**Pull request by ftessier**
Provide a sample EGSnrc application which scores a pulse-height distribution in a set of regions specified by a label. Included is a sample input to model a germanium detector inside a lead shield, irradiated by a collimated
point source. Also included are egs_view settings and a viewer screen shot for reference.

ge-example

### CODE REVIEW

**Comment by mainegra** [comment]
@ftessier, I wonder if egs_phd would be a more appropriate name to differentiate it from a differential fluence calculation.

**Comment by ftessier** [comment]
Good idea, I will change the name to `egs_phd`.

File: HEN_HOUSE/pegs4/density_corrections/compounds/cellulose.density [hunk 1]
```diff
@@ -0,0 +1,32 @@
+  cellulose
```
Comment (ftessier) [inline HEN_HOUSE/pegs4/density_corrections/compounds/cellulose.density]: Can you include a short reference? I find that in that with time this is often useful: knowing where the author got the data, or how it was derived. Same comment for the silicon dioxide file

Comment (elgoub) [reply HEN_HOUSE/pegs4/density_corrections/compounds/cellulose.density]: how can  get efficiencies  for this detectors in specific energy ??

**Review by blakewalters — APPROVED** [review]

**Comment by ftessier** [comment]
Rebased on develop.

**Comment by ftessier** [comment]
Tweak commits, fix `finalize_egs_foruser`, adjust format of HPGe sample input file. No functional changes.

**Comment by ftessier** [comment]
Squash the commit that only added better descriptions in the supplied density correction files.

**Comment by gweshitt** [comment]
I tested the egs_phd example. The output file ge-example-spectrum.dat contains the correct Emin, Emax and binning, but the 2nd and 3rd columns containing the spectrum data are all zeros. I've looked over the egs_phd.cpp file, but I can't see how this would happen.

**Comment by mjukdisk** [comment]
In the icrm-hpge-example.egsinp you use a density correction file called icrm_dirt. Could you upload that? alternatively post the "formulae" to create it using ESTAR?

**Comment by rtownson** [comment]
Thanks @mjukdisk, see PR #721

**Pull request merged**